packages/client/e2e/fixtures.ts

```diff
@@ -143,7 +143,7 @@ export const test = base.extend<Fixtures>({
         mainPage: mainWindow,
         blotterPO: new CreditBlotterPageObject(mainWindow),
         newRfqPO: new CreditNewRfqPageObject(mainWindow),
-        rfqsPO: new CreditRfqTilesPageObject(mainWindow),
+        rfqsPO: new CreditRfqTilesPageObject(mainWindow, workerInfo),
```

Comment:
I am strongly inclined to pass `workerInfo` into POs as a standard .. we will need this kind of thing elsewhere (FX and mobile, I'd think)